src/uu/ls/Cargo.toml

```diff
@@ -15,9 +15,9 @@ edition = "2021"
 path = "src/ls.rs"
 
 [dependencies]
+ansi-width = "0.1.0"
 clap = { workspace = true, features = ["env"] }
```

Comment:
The fact that every entry except this one says `workspace = true` and doesn't provide a specific version number makes me wonder why it's the other way around here.

Reply:
Because I forgot 😅